src/App.js

```diff
@@ -327,6 +332,8 @@ class App extends Component {
           store.dispatch(refreshAccessToken())
         }, 1000 * 60 * 50)
       }
+    } else {
+      await store.dispatch(preLoginActions())
```

Comment:
don't we need to fetch ethContract for authenticated users?

Reply:
It's done inside the postLoginPreparation function for authenticated users